### DISCUSSION TITLE
All checks have failed: we cannot make changes to the indicators

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1814] [Q&A]
We cannot make changes to the indicators 16-1-1, 14-3-1, 9-5-1.
Please let me know what errors need to be corrected.

**Comment by christoph-LSN** [comment]
@ukraine-sdg could you please link your site and data repo?

**Comment by christoph-LSN** [comment]
@ukraine-sdg not sure but these lines are no in the other indicator-config files.... so may be deleting them might help

https://github.com/ukraine-sdg/data/blob/c1f9190582ab091cbcd2ee7ff0a876e39cf153cc/indicator-config/14-3-1.yml#L1-L28

the failing test mentions this file:  indicator-config/14-3-1.yml

**Comment by brockfanning** [comment]
@ukraine-sdg Are you still unable to update those indicators?

**Comment by ukraine-sdg** [comment]
Thanks for your help, removed
the redundant information and the updates are displayed.
